climpred/tests/test_toy.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import xarray as xr
+from climpred.prediction import compute_perfect_model
+from climpred.toy import create_initialized, run_skill_for_ensemble
+from climpred.tutorial import load_dataset
+from xarray.testing import assert_allclose
+
+# standard parameters; can be changed
+lead = xr.DataArray(np.arange(0, 20, 1), dims='lead')
+lead['lead'] = lead.values
```

Comment:
Isn't this redundant? `xarray` will automatically add this coordinate since it's the same as running `np.arange(...)` on your dim size.